upload.py

```diff
@@ -149,6 +149,11 @@ def check_missing_data(asset_type, props):
     if len(props.name) > 40:
         write_to_report(props, f'The name is too long. maximum is 40 letters')
 
+    if props.category == 'NONE' or \
```

Comment:
I suggest to split this condition into three separate conditions, this is really hard to read.